app/Models/Auth.php

```diff
@@ -269,7 +269,23 @@ public static function getCredentialsFromCookie() {
 		}
 
 		$credentials = @file_get_contents($token_file);
-		return $credentials === false ? array() : explode("\t", $credentials, 2);
+		if ($credentials !== false && self::renewCookie($token)) {
+			return explode("\t", $credentials, 2);
+		}
+		return [];
+	}
+
+	private static function renewCookie($token) {
+		$token_file = DATA_PATH . '/tokens/' . $token . '.txt';
+		if (touch($token_file)) {
+			$conf = Minz_Configuration::get('system');
+			$limits = $conf->limits;
+			$cookie_duration = empty($limits['cookie_duration']) ? 2592000 : $limits['cookie_duration'];
```

Comment:
I'm still very frequently annoyed by this imho extremely low default value. Almost every time I start Steam, I have to log in again. Excuse me for only playing a game once every month or two…

Speaking as a user, I don't think this should be an "instead" at all, but an "in addition to." I'll leave the exact value in the middle, but I'll repeat that I'd prefer to see at the very least 3 months by default.

Reply:
Ok @Frenzie , let's re-open https://github.com/FreshRSS/FreshRSS/pull/3170 and change it to 3 months (could you patch it?), in addition of the current PR